db/products-controllers.ts

```diff
@@ -0,0 +1,94 @@
+import Product from './products';
+import * as productValidation from '../config/products-validation';
+import {logger} from '../config/logger';
+
+class ProductsControllers {
+    async getProducts(ctx) {
```

Comment:
any route function get second parameter called next. what is the purpose of this parameter? and why do you think you need to use it here